scripts/all-tests.sh

```diff
@@ -15,27 +15,30 @@ set -e
 #   Possible values are:
 #
 #       - "quality": Run the quality (pep8/pylint) checks
-#       - "unit": Run the JavaScript and Python unit tests
-#            (also tests building the Sphinx documentation,
-#             because we couldn't think of a better place to put it)
+#       - "lms-unit": Run the LMS Python unit tests
+#       - "cms-unit": Run the CMS Python unit tests
+#       - "commonlib-js-unit": Run the JavaScript tests and the Python unit
+#         tests from the common/lib directory
 #       - "lms-acceptance": Run the acceptance (Selenium) tests for the LMS
 #       - "cms-acceptance": Run the acceptance (Selenium) tests for Studio
 #       - "bok-choy": Run acceptance tests that use the bok-choy framework
 #
 #   `SHARD` is a number (1, 2, or 3) indicating which subset of the tests
-#       to build.  Currently, "lms-acceptance" and "bok-choy" each have two
-#       shards (1 and 2), "cms-acceptance" has three shards (1, 2, and 3),
-#       and all the other test suites have one shard.
+#       to build.  Currently, "lms-unit" has two shards (1 and 2),
```

Comment:
I wonder if we just remove these lines...we'd have to update them again when we have more shards.